docs/octopus-cloud/migrations.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Migrating from self-hosted to Octopus Cloud
+position: 40
+description:  Migrating from self-hosted to Octopus Cloud.
+---
+
+If you have an existing self-hosted Octopus server, and are looking to switch to using Octopus Cloud, you may want to migrate all of your existing projects, variables, history and configuration through to your new cloud instance. Due to some specific differences between the two platforms, we only support this through an entire database backup and restore, to ensure that we don't miss any key dependencies.
+
+:::warning
+**Note:** our existing [Migration API](/docs/api-and-integration/migration-api/index.md) is **not supported** for migrations to cloud instances at this time.
+:::
+
+## Before you start
+
+This migration process is intended as a one-off operation to lift and shift everything across to an Octopus Cloud instance. Before you schedule time with our team to do the migration, please keep in mind:
+
+- there will be an outage of 1-2 business days while we migrate your data through to a running cloud instance
+- we will need a number of data files to do the migration on our end, so please schedule some time to export the necessary files and artifacts
+- exports will need to be generated from a **minimum** version of 2018.10, so please make sure you are running this version or higher before exporting the database
```

Comment:
I've just guessed a version of `2018.10` here, as that is our first LTS cut. Happy for this to be changed to whatever though

Reply:
I think this doc should leave some room for us to dictate the version. "You will need to upgrade your self-hosted instance to a specific version in preparation for the migration."

Reply:
I've reworded this to be less specific